README.md

```diff
@@ -237,6 +237,36 @@ Test project /path/to/Beman.Optional26/.build
 No tests were found!!!
 ```
 
+#### Pre-Commit for Linting
```

Comment:
Would you like to move this section outside the root README? It's useful for Beman devs, but not for users.

I would move it into a file `docs/dev-lint.md` (or whatever).